README.md

```diff
@@ -10,6 +10,14 @@ Python Battery Mathematical Modelling solves continuum models for batteries, usi
 
 ## How do I use PyBaMM?
 
+The easiest way to use PyBaMM is to run a constant-current discharge with a model of your choice:
+```
+$ python3 pybamm/simulation.py model_name
+```
+where model_name is the name of one of the [model classes](pybamm/models/). 
+
```

Comment:
might want to also say something about how users can customise simulations using the base mode/parameter/discretisation/solver classes here